### ISSUE TITLE
Why Unstated?

### ISSUE THREAD

**Issue by mbrahimi02** [jamiebuilds/unstated-next#90]
I'm just failing to understand why use unstated. The implementation looking at the source doesn't seem to do anything to add to the existing context API, no memoized selectors, no atomic API. So why is unstated so popular and what does it add?

**Comment by y-hiraoka** [comment]
@mbrahimi02 
I think it's popular because it has very simple APIs. But wouldn't it be more convenient to have memorized selectors?
I have developed a library called `unreduxed` that has almost the same API as` unstated-next`. This allows the `useConatiner` hook to receive a selector function and suppresses extra re-rendering.

Please use it by all means;)

**Comment by AjaxSolutions** [comment]
> I'm just failing to understand why use unstated. The implementation looking at the source doesn't seem to do anything to add to the existing context API, no memoized selectors, no atomic API. So why is unstated so popular and what does it add?

I agree.  This library is just a wrapper around the React context api which doesn't add any new functionality.

I like these three state management libraries better: Zustand, Valtio and Jotai.

https://github.com/pmndrs

**Comment by jmayergit** [comment]
>I like these three state management libraries better: Zustand, Valtio and Jotai.

> https://github.com/pmndrs

I looked into one of these libraries, Zustand, and the setup for Next.js is HUGE

https://github.com/vercel/next.js/blob/master/examples/with-zustand/lib/store.js

**Comment by Sti2nd** [comment]
Agreed. Use React Context; it has almost the exact same API as unstated-next. I guess unstated-next only exists because unstated existed, and _maybe_ they look somewhat alike 🤷🏿
